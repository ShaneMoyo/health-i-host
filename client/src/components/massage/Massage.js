import React from 'react';
import Promo from '../utils/Promo';
import {  mdiHumanHandsup } from '@mdi/js'

export default function Massage() {
  const text = [
    'Massage',
    'An intuitive art form rooted in scientific and experiential hands on healing to allow the body to move freely in its surroundings.',
    'Stress can take a toll on your body, and even though your body works hard to keep up, it needs help. Keeping your body running efficiently should be high on your to-do list and regular massage is key to operating at peak efficiency. Keeping your body in optimal working condition with routine massage - along with Rapid Tension Relief and Total Body Stretch - is easy at any Massage Envy franchised location.'
  ]

  return <div>
            <Promo header1={text[0]} header2={text[1]} header3={text[2]} icon={mdiHumanHandsup} color='primary'/>
          </div>
}
