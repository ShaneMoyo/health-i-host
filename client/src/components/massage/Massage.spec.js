import React from 'react';
import { shallow } from 'enzyme';
import Massage from './Massage';
describe('Massage', () => {
  it('should render correctly massage component', () => {
    const component = shallow(<Massage />);

    expect(component).toMatchSnapshot();
  });
});
