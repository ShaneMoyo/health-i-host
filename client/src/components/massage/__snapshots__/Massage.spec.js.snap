// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Massage should render correctly massage component 1`] = `
<div>
  <Promo
    color="primary"
    header1="Massage"
    header2="An intuitive art form rooted in scientific and experiential hands on healing to allow the body to move freely in its surroundings."
    header3="Stress can take a toll on your body, and even though your body works hard to keep up, it needs help. Keeping your body running efficiently should be high on your to-do list and regular massage is key to operating at peak efficiency. Keeping your body in optimal working condition with routine massage - along with Rapid Tension Relief and Total Body Stretch - is easy at any Massage Envy franchised location."
    icon="M5,1C5,3.7 6.56,6.16 9,7.32V22H11V15H13V22H15V7.31C17.44,6.16 19,3.7 19,1H17A5,5 0 0,1 12,6A5,5 0 0,1 7,1M12,1C10.89,1 10,1.89 10,3C10,4.11 10.89,5 12,5C13.11,5 14,4.11 14,3C14,1.89 13.11,1 12,1Z"
  />
</div>
`;
